lib/src/widgets/custom_card.dart

```diff
@@ -48,7 +48,7 @@ class CustomCard extends StatelessWidget {
                 children: [
                   Padding(
                     padding: const EdgeInsets.all(12.0),
-                    child: Image.network(
+                    child: Image.asset(
```

Comment:
Is there a particular reason to change where the image is fetched from?

Reply:
When I tested a widget,  file test war error. It's said You must have httpClient . So, I change it to test the screens.

Reply:
Can I change it or not ?

Reply:
If that is the case then you cannot. Look up how to test widgets that rely on network calls 😉 .

Additionally, This widget may break if the API dies or the device fails to fetch the content. Look up how to remedy that.
```suggestion
                    child: Image.asset(
```

Reply:
Yah, I will change this code like before. After that, I will solve problem for the test.

Reply:
I have just committed this file. You can review that.